foo = ['a', 'b', 'c']
print(foo[3])      # will this result in an error?

# results in an IndexError